hibernate-core/src/main/java/org/hibernate/dialect/OracleServerConfiguration.java

```diff
@@ -46,39 +57,82 @@ public int getDriverMinorVersion() {
 	}
 
 	public OracleServerConfiguration(boolean autonomous, boolean extended) {
-		this( autonomous, extended, 19, 0 );
+		this( autonomous, extended, false, 19, 0 );
+	}
+
+	public OracleServerConfiguration(boolean autonomous, boolean extended, boolean applicationContinuity) {
+		this( autonomous, extended, applicationContinuity, 19, 0 );
 	}
 
 	public OracleServerConfiguration(
 			boolean autonomous,
 			boolean extended,
+			boolean applicationContinuity,
 			int driverMajorVersion,
 			int driverMinorVersion) {
 		this.autonomous = autonomous;
 		this.extended = extended;
+		this.applicationContinuity = applicationContinuity;
```

Comment:
```suggestion
			int driverMajorVersion,
			int driverMinorVersion) {
		this( autonomous, extended, false, driverMajorVersion, driverMinorVersion );
		
	public OracleServerConfiguration(
			boolean autonomous,
			boolean extended,
			boolean applicationContinuity,
			int driverMajorVersion,
			int driverMinorVersion) {
```

Reply:
We must keep previous constructor signatures for backwards compatibility. Only create a new one with an additional `boolean applicationContinuity` argument and let others delegate to this one.